tests/resources/custom.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" ?>
 <clickhouse>
     <listen_host>::</listen_host>
-    <listen_host>0.0.0.0</listen_host>
```

Comment:
https://clickhouse.com/docs/en/operations/server-configuration-parameters/settings/#server_configuration_parameters-listen_host

`<listen_host>::</listen_host>` already means to listen on all hosts. No need to specify `0.0.0.0` as well (and is double specified).
Moved port definitions closer to listen host for easier spotting.